src/borg/remote.py

```diff
@@ -1,5 +1,6 @@
 import errno
 import fcntl
+import inspect, functools
```

Comment:
also most new functions and variables are camelCase, while snake_case is the usual choice here.